.gitignore

```diff
@@ -46,7 +46,7 @@ tsan/
 asan/
 ubsan/
 opt/
-build/
+build*/
```

Comment:
Ack.  You know you can build inside `build/Ubuntu16.04`? (notice the extra / ).

Just do:

```
mkdir -p build/Ubuntu16.04
cd build/Ubuntu16.04
../../configure 
```